galasa-ui/src/components/test-runs/OverviewTab.tsx

```diff
@@ -49,6 +49,10 @@ const OverviewTab = ({ metadata }: { metadata: RunMetadata }) => {
         title={`${translations("test")}:`}
         value={metadata?.testName}
       />
+      <InlineText
+        title={`${translations("package")}:`}
+        value={metadata?.package}
+      />
```

Comment:
Minor - might be worth updating a unit test to check that the package is being rendered in the overview tab?